python/Ganga/Utility/Runtime.py

```diff
@@ -257,12 +257,14 @@ def loadPlugins(environment):
             logger.error('problems with loading Named Templates for %s', n)
             logger.error('Reason: %s' % str(err))
 
-    for r in allRuntimes.values():
+    for n, r in zip(allRuntimes.keys(), allRuntimes.values()):
         try:
             r.loadPlugins()
         except Exception as err:
-            logger.error('problems with loading Named Templates for %s', n)
+            from Ganga.Core.exceptions import GangaException
```

Comment:
I may be missing something but looking at the code you don't pull in anything from the `__init__` of `Ganga.Core` and in most other places in the code base I've seen the exception imports are at the top of the code. Is there a circular import problem?